### ISSUE TITLE
Process dialog - weird behaviour

### ISSUE THREAD

**Issue by SirRickie** [umbrellaplug/umbrellaplug.github.io#402]
Kodi 21.1/Wn11/Umbrella 6.7.0/skin Arctic Horizon/long time Umbrella user.
Issue : with Umbrella and via the context menu, I add a TV show to the library. When I launch an episode, the process dialog (scraping and resolving) is displayed as **background** (top right corner), although in settings/source options, the process dialog is set as **full-screen**. I think this has started with latest update 6.7.0. How can I fix this, so that the dialog as always in full-screen mode (like it always used to be).
Strange enough, when I launch an episode, which is **not** in the library, then the dialog is on full-screen.
Tnx for you help.

**Comment by umbrellaplug** [comment]
I will try to duplicate this behavior. Can you post your log?

**Comment by SirRickie** [comment]
Additional info : when I launch an episode of a TV show, which is added to the library via TMDBHelper and I launch it with TMDBH with Umbrella as a player, then the process dialog is indeed in full-screen mode (not background mode). Weird. But I prefer to add shows in the library with Umbrella, it's faster. In my log, I will do both : I launch an episode of "Alaska Daily" with TMDBH and Umbrella as a player (will be full-screen mode) and afterwards I launch an episode of "We were the lucky ones" with Umbrella, not TMDBH (will be background mode). Log will follow as soon as I get home.

**Comment by SirRickie** [comment]
Enclosed the log. I have to correct my previous comment. I wrote that, if I launch an episode with TMDBH and with player Umbrella, the dialog is in full-screen mode, but that was not correct, the dialog is also in background mode.
In the log, I have launched with Umbrella (not TMDBH) an episode of “we were the lucky ones” and the dialog was in background mode (although set in settings to full-screen mode). Hope you find the issue.
https://paste.kodi.tv/aquhuluxar.kodi
